Resources/Maps/_Mono/Shuttles/framework.yml

```
@@ -0,0 +1,541 @@
+meta:
+  format: 7
+  category: Grid
+  engineVersion: 247.2.0
+  forkId: ""
+  forkVersion: ""
+  time: 03/30/2025 07:16:15
+  entityCount: 75
+maps: []
+grids:
+- 2
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 0

(indentation)